src/main/java/anto/TaskList.java

```diff
@@ -0,0 +1,218 @@
+package anto;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * TaskList class represents list of all current tasks and handles all actions
+ * related to tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    /**
+     * Creates a new TaskList class.
+     *
+     * @param taskList Initial ArrayList of tasks.
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Add task given to arraylist.
+     *
+     * @param newTask Task to be added.
+     * @throws AntoException Throws anto exception if there is an IO Exception
+     */
+    public void addToList(Task newTask) throws AntoException {
+        this.taskList.add(newTask);
+
+        String relativePath = "data/anto.txt";
+        Path absolutePath = Paths.get(relativePath).toAbsolutePath();
+        File antoFile = absolutePath.toFile();
+
+        try {
+            FileWriter writer = new FileWriter(antoFile, true);
+            if (newTask instanceof Todo) {
+                writer.write(String.format("\nT | %s | %s", newTask.isDone ? "1" : "0", newTask.description));
+            } else if (newTask instanceof Deadline) {
+                writer.write(String.format("\nD | %s | %s | %s", newTask.isDone ? "1" : "0", newTask.description, (
+                        (Deadline) newTask).by.format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"))));
+            } else if (newTask instanceof Event) {
+                writer.write(String.format("\nE | %s | %s | %s | %s", newTask.isDone ? "1" : "0", newTask.description, (
+                        (Event) newTask).from.format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm")), (
+                        (Event) newTask).to.format(DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm"))));
+            }
+            writer.close();
+        } catch (IOException e) {
+            throw new AntoException("OOPS!!! IOException");
+        }
+    }
+
+    /**
+     * Returns current array list of tasks.
+     *
+     * @return ArrayList of tasks.
+     */
+    public ArrayList<Task> getStorage() {
```

Comment:
should this be named as something else? Could be mistaken for returning a Storage instance